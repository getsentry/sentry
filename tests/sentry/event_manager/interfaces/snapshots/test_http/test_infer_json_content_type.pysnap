---
created: '2019-03-14T17:12:35.734267Z'
creator: sentry
source: tests/sentry/event_manager/interfaces/test_http.py
---
errors: null
to_json:
  data:
    foo: bar
  inferred_content_type: application/json
  url: http://example.com/
