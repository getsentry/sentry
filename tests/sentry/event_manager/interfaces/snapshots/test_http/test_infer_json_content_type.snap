errors: null
to_json:
  data: {foo: bar}
  inferred_content_type: application/json
  url: http://example.com/
